engine/src/test/scala/engine/workflow/ValidateActorSpec.scala

```diff
@@ -0,0 +1,177 @@
+                if (!validationException.message.contains("Workflow input processing failed.")) fail("Message coming from validation exception is not the expected one.")
+                if (validationException.errors.size != 1) fail("Number of errors coming from validation exception are not the expected ones.")
+                if (!validationException.errors.head.contains("Workflow contains invalid inputs JSON")) fail("Message from error nr 1 in validation exception is not the expected one.")
+            }
+          case unknown => fail(s"Response is not equals to the expected one. Response: $unknown")
+        }
+      }
+      verify(backendMock, times(1)).assertWorkflowOptions(WorkflowOptions.fromJsonString(validOptionsJson).get)
+    }
+
+    "return ValidationFailure when there are invalid workflow inputs coming from a workflow inputs JSON file" in {
+      within(1000 millis) {
+        validateActor ! ValidateWorkflow(validWdlSource, Some(invalidInputJson), Some(validOptionsJson))
+        expectMsgPF() {
+          case validationFailure: ValidationFailure =>
+            validationFailure.reason match {
+              case validationException: ValidationException =>
+                if (!validationException.message.contains("Workflow input processing failed.")) fail("Message coming from validation exception is not the expected one.")
+                if (validationException.errors.size != 1) fail("Number of errors coming from validation exception are not the expected ones.")
+                if (!validationException.errors.head.contains("Required workflow input 'hello.hello.addressee' not specified.")) fail("Message from error nr 1 in validation exception is not the expected one.")
+            }
+          case unknown => fail(s"Response is not equals to the expected one. Response: $unknown")
+        }
+      }
+      verify(backendMock, times(1)).assertWorkflowOptions(WorkflowOptions.fromJsonString(validOptionsJson).get)
+    }
+
+    "return ValidationFailure when there are invalid runtime requirements coming from WDL file" in {
+      when(backendMock.backendType).thenReturn(BackendType.JES)
+      within(1000 millis) {
+        validateActor ! ValidateWorkflow(invalidWdlSource, Some(validInputsJson), Some(validOptionsJson))
+        expectMsgPF() {
+          case validationFailure: ValidationFailure =>
+            validationFailure.reason match {
+              case validationException: ValidationException =>
+                if (!validationException.message.contains("Workflow input processing failed.")) fail("Message coming from validation exception is not the expected one.")
+                if (validationException.errors.size != 1) fail("Number of errors coming from validation exception are not the expected ones.")
+                if (!validationException.errors.head.contains("Failed to validate runtime attributes.")) fail("Message from error nr 1 in validation exception is not the expected one.")
```

Comment:
I'm just checking part of the error message. The part of the message I'm checking only comes from the function that validates the runtimeReq.